typescript/db/deploy-vercel.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [[ -n $MIGRATE_DATABASE ]]; 
+ then POSTGRES_EXTERNAL_URL=$MIGRATE_DATABASE yarn deploy && POSTGRES_EXTERNAL_URL=$POSTGRES_EXTERNAL_URL; 
+ else echo 'Skipping migration. No $MIGRATE_DATABASE' && echo $MIGRATE_DATABASE; 
```

Comment:
Do we want to keep the second `echo`?

Reply:
I left it so that we can more easily track if a migration is happening or not, but if you feel it's not used I can remove it

Reply:
No, that's ok. I wanted to know if this was only to debug it when you implemented this. But you're right, it may be useful later on.